runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -106,12 +112,34 @@ export default class ExcelActionHandler extends StatefulActionHandler {
         cell.Value = input.value;
     }
 
+    async setCells(
+        input: ExcelSetCellsActionInput
+    ): Promise<void> {
+        this.checkIsObject(input.targetExcelStructure);
+
+        const optionalWorksheet = input?.worksheet;
+        const openedWorksheet = this.session.ActiveSheet.name;
+
+        if (optionalWorksheet) this.session.Worksheets(optionalWorksheet).Activate();
+        for (const [coordinate, cellValue] of Object.entries(input.targetExcelStructure)) {
+            const cell = this.session.ActiveSheet.Range(coordinate);
+            cell.Value = cellValue;
+        }
+        if (optionalWorksheet) this.session.Worksheets(openedWorksheet).Activate();
+    }
+
```

Comment:
maybe we should also consider an approach that allows to fill Excel file with a range of cells (located somewhere next to each other), without directly indicating the place for each cell separately (but for example, indicating the location of only the first cell in the range)

Reply:
done

Reply:
both options possible
1. Array of arrays (this is how it's done in RPA Framework) which allows to set cells just like in excel (optionally can specify starting cell e.g. D8:
<img width="294" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/d57ee1c3-2d8c-4a2b-b75a-a4faab9ed103">
2. Object with precisely defined cooridnates and values { A2: "value, ......... }